opencsp/common/lib/camera/ImageAcquisitionAbstract.py

```diff
@@ -57,24 +59,25 @@ def _check_saturated(im):
 
         # Check if exposure is set
         max_iters = int(np.ceil(np.log2(exposure_values.size)) + 1)
-        for i in range(max_iters):
+        for _ in range(max_iters):
             # Get next exposure index to test
-            idx = _get_next_exposure_idx()
+            idx_1 = _get_next_exposure_idx()

```

Comment:
Not that this has to be done on this PR, but the description for the property `shutter_cal_values` should be updated to indicate that the automatic calibration works better when the returned exposure time values are in a linear and even spacing.

Reply:
Good point (yay documentation)! I changed it. I figured I might as well do it here since we're thinking about it if you're OK with it.

Reply:
I just took a look at the new docstring. It looks great!